i18n/en.json

```diff
@@ -24,6 +25,8 @@
 	"categories": "Categories",
 	"categories-panel-title": "Categories filter",
 	"categories-description": "Enter categories (with wiki names) to limit metrics to pages in those categories",
+	"categories-description-tooltip": "Learn about Categories filtering",
```

Comment:
looks like we're missing qqq for this message

Reply:
Fixed.